draft-barnes-acme.md

```diff
@@ -1280,7 +1280,7 @@ If the server presents a certificate matching all of the above criteria, then th
 
 ## Proof of Possession of a Prior Key
 
-The Proof of Possession challenge verifies that a client possesses a private key corresponding to a server-specified public key, as demonstrated by its ability to correctly sign server-provided data with that key.
+The Proof of Possession challenge verifies that a client possesses a private key corresponding to a server-specified public key, as demonstrated by its ability to sign with that key.
```

Comment:
server-specified? I think I know where you are getting at, but it's not like the server can choose a random key.